bpython/curtsiesfrontend/repl.py

```diff
@@ -408,13 +409,16 @@ def __init__(
         # Like self.history, lines are unwrapped, uncolored, and without prompt.
         # Entries are tuples, where
         #   - the first element the line (string, not fmtsr)
-        #   - the second element is one of 2 global constants: INPUT or OUTPUT 
-        #     indicating the type of line
+        #   - the second element is one of 2 global constants: "input" or "output",
         self.all_logical_lines = []
+        
+        # This is to use "input" or "output" for all_logical_lines tuples without actually 
+        # having to know the string.
+        class lineTypeTranslator(Enum):
```

Comment:
I'd move this out to the global scope so that other code can access it without needing to use self.lifeTypeTranslator, and use TitleCase instead of camelCase

Reply:
Got it. I added it as a class in bpython.repl, and now curtsiesfrontend.repl imports it.